src/ArethusaConfig.js

```diff
@@ -3,7 +3,7 @@ const defaultConfig = {
     debug: false,
     showKeys: false,
     chunkParam: 'chunk',
-    auxConfPath: `${process.env.PUBLIC_URL}/arethusa/configs`,
+    auxConfPath: '/arethusa/configs', // TODO
```

Comment:
Do you know if there is any way to make this path relative to the `remoteUrl` passed to Arethusa? If not, then I think each project will have to modify the config.